testing/typescript/data/custom_store.ts

```diff
@@ -0,0 +1,57 @@
+import $ from 'jquery';
```

Comment:
Is all this stuff really required? What exactly checks the test?

Reply:
I agree that this is not rational - we only need ajax for data request. However, jquery is exported as a whole chunk. We can use, for example, http://projects.jga.me/jquery-builder or https://github.com/jquery/jquery#how-to-build-your-own-jquery. Do we really need to make our own special cropped custom JQuery bundle?

Reply:
Sry, I was not enough specific, by 'all stuff' I mean creating a class, defining several methods, etc. Can't it be reduced to 3-5 lines?

Reply:
Reduced by 30%